### ISSUE TITLE
[Feature request] when adding friends to the groupchat, notify if they are already in it

### ISSUE THREAD

**Issue by griffondor** [qTox/qTox#1233]
I would like to report a bug : when adding twice the same person who is already in the groupoid is no notification telling us : that person is already in the group

**Comment by Hdram** [comment]
Description:

Info:
OS: Win Enterprise Server 2003
Hardware: Dell Dimension
…

Reproducible: Always 

Steps to reproduce:
1. Open Settings
2. Click on "Save files" (to change location)
3. 
…

What happened:Nothing happens

What should happen:The box should allow me to type a new location

Additional info:
If I tick the "Autoaccept" box and then left click on the location box a "Choose an autoaccept directory" box appears, which allows me to select a new location, Then when I un-tick the "autoaccept" the new location remains in the location box.
However when I then save a file it still saves to the original "default" location which is "C:/Program Files/qTox/bin

**Comment by agilob** [comment]
@Hdram I don't understand your problem.
`Save files` is a part of the checkbox `autoaccept and save files`
[That's how is should behave.](http://upload.agilob.net/qtox_filechooser.webm). Do you have different behaviour or options?

**Comment by Hdram** [comment]
@agilob Yes autoaccept and save files. Well then it seems I misunderstood this arrangement.
So if that destination box only refers to autoaccept (which I don't want to use, since I don't want to accept every file I'm offered ) assuming that that's what autoaccept means. Is there some other way to change where files are saved when manually accepted, other than every single time having to browse from "C:/Program Files/qTox/bin" to e.g."F:/Downloads/ qTox saved files"?
Similarly when I want to send a file, once I click the paper clip I'm always brought to "C:/Program Files/qTox/bin" since qTox doesn't remember the last folder I browsed to, like most applications do.

**Comment by zetok** [comment]
With current (broken) groupchats it would be doable.

Things are going to change though, and with new groupchats it will not be possible.

What's changing?

Current groupchats simply use own, long-term public key. It would be possible to compare list of public keys in groupchat to the public keys of friends.

New groupchats generate new keypair for every new groupchat that is being joined. This would render comparing public keys pointless.

Because of that, I think that this issue should be closed.
